serverless-sample/app/aws/Makefile

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
To validate the env vars add this to the top of the Makefile:
ifndef GCR_REPO
$(error GCP_REPO is undefined)
endif
ifndef tag
tag=latest
endif